protocol/0044-LIME-lp_mechanics.md

```diff
@@ -341,11 +341,19 @@ In the case of spot markets it will be transferred into the network treasury for
   - LP can cancel their commitment without incurring penalties (<a name="0044-LIME-053" href="#0044-LIME-053">0044-LIME-053</a>)
 - For a market that is in continuous trading and a single LP has committed liquidity:
   - The LP can cancel their commitment at any time (though this may involve incurring a penalty) (<a name="0044-LIME-060" href="#0044-LIME-060">0044-LIME-060</a>) for spot (<a name="0044-LIME-056" href="#0044-LIME-056">0044-LIME-056</a>)
+  - The LP can increase their commitment at any time without incurring fee (<a name="0044-LIME-061" href="#0044-LIME-061">0044-LIME-061</a>) for spot (<a name="0044-LIME-062" href="#0044-LIME-062">0044-LIME-062</a>)
+  - When LP increases their commitment higher than their account balance are rejected and old provision persists (<a name="0044-LIME-063" href="#0044-LIME-063">0044-LIME-063</a>) for spot (<a name="0044-LIME-064" href="#0044-LIME-064">0044-LIME-064</a>)
```

Comment:
This isn't clear about what's being compared to what.

Reply:
LP which decreases the commitment are charged as factor of market.liquidity.earlyExitPenalty and when the commitment is increased from x to x+1000, they are not charged any fee.

Reply:
However, when LP tries to increase the commitment greater than their available fund are rejected. Lets say LP got 1000 units in General and 1000 units in Margin and 100 unit in Bond accoutn. When they try to increase their commitment from 100 units to 3000 units , it fails .

Reply:
What you say may be fine but the AC needs to stand on its own (at least for most people) - the explanatory comments left here won't be visible to someone just reading the spec.